effects/steal_effect.py

```diff
@@ -1,14 +1,13 @@
 from game import Game
 from effects.effect import Effect
 from citizens.citizen import Citizen
-from citizens.spy import Spy
 import user_interaction
 import utils
 
 
 class StealEffect(Effect):
 
-    def __init__(self, name: str, game: Game, creator: Citizen) -> None:
+    def __init__(self, name: str, game: Game, creator: Citizen):
         Effect.__init__(self, name, game, creator)
 
     def activate(self):
```

Comment:
Inside of all activate methods we should change effect status to "EffectStatus.ACTIVATE".
And inside all resolve methods we should change effect status to "EffectStatus.FINISHED"
!!!
important. Should be discussed